src/controllers/streamingsettings.js

```diff
@@ -30,24 +31,6 @@ define(["jQuery", "libraryMenu", "loading"], function ($, libraryMenu, loading)
     }
 
     $(document).on("pageinit", "#streamingSettingsPage", function () {
-        var page = this;
-        $("#btnSelectTranscodingTempPath", page).on("click.selectDirectory", function () {
```

Comment:
why is this no longer used? is transcoding path no longer controllable?

Reply:
It's controllable in a separate page, this must have been left over from some other time.